docs/topics/js/js-overview.md

```diff
@@ -29,6 +29,23 @@ To learn more about the available features in the Kotlin/JS IR compiler and how
 Modern web development benefits significantly from frameworks that simplify building web applications.
 Here are a few examples of popular web frameworks for Kotlin/JS written by different authors:
 
+### Kobweb
+
+_Kobweb_ is an opinionated Kotlin framework for creating websites and web apps. It leverages [Compose HTML](https://github.com/JetBrains/compose-multiplatform?tab=readme-ov-file#compose-html) and
+live-reloading for fast development. Inspired by [Next.js](https://nextjs.org/), Kobweb promotes a standard structure for adding widgets, layouts,
+and pages.
+
+Out of the box, Kobweb provides page routing, light/dark mode, CSS styling, Markdown support, backend APIs, and more features.
+It also includes a UI library called Silk, a set of versatile widgets for modern UIs. 
+
+Kobweb also supports site export, generating page snapshots
+for SEO, and automatic search indexing. Additionally, Kobweb makes it easy to create DOM-based UIs that efficiently update in response to state changes.
+
+Visit the [Kobweb](https://kobweb.varabyte.com/) repository for documentation and examples.
```

Comment:
Nit: this is more of a landing page than a repository, but I'm not sure most people will notice?

So maybe

Visit the [Kobweb site](https://kobweb.varabyte.com/) for documentation and examples.